hack/deploy.sh

```diff
@@ -79,6 +79,8 @@ function status(){
       ${CMD} describe pod -n ${HCO_NAMESPACE} ${PNAME} || true
       ${CMD} logs -n ${HCO_NAMESPACE} ${PNAME} --all-containers=true || true
     done
+    HCO_POD=$( ${CMD} get pods -l "name=hostpath-provisioner-operator" -o name)
```

Comment:
you are selecting the HPP pod, not HCO

Reply:
Thanks. need to be fixed on master as well